neuralmonkey/decoders/transformer.py

```diff
@@ -115,7 +116,8 @@ def __init__(self,
 
         self.encoder_states = get_attention_states(self.encoder)
         self.encoder_mask = get_attention_mask(self.encoder)
-        self.dimension = self.encoder_states.get_shape()[2].value
+        self.dimension = (
+            self.encoder_states.get_shape()[2].value)  # type: ignore
```

Comment:
Tady by to možná chtělo něco asercí, že sedí dimenze modelu, embedingy, vyhodit warning, když `ff_hidden_size` je menší než model dimension.

Reply:
to se mi teď nechce dělat...